cmd/kratos/internal/proto/add/add.go

```diff
@@ -63,7 +65,11 @@ func javaPackage(name string) string {
 }
 
 func serviceName(name string) string {
-	return export(strings.Split(name, ".")[0])
+	return UnderscoreToUpperCamelCase(strings.Split(name, ".")[0])
 }
 
-func export(s string) string { return strings.ToUpper(s[:1]) + s[1:] }
+func UnderscoreToUpperCamelCase(s string) string {
```

Comment:
No need to export this.  `UnderscoreToUpperCamelCase` -> `toUpperCamelCase` maybe better.

Reply:
done.